corehq/apps/hqwebapp/static/hqwebapp/util/proptable/style.css

```diff
@@ -3,6 +3,10 @@
     margin-right: 18px;
 }
 
+.sink {
```

Comment:
I think you're right. You could have a float in a float.

("sink" is an improvement on my first name, "bee" (because butterflies are all about floating, [not bees][1].) :smiley: )

  [1]: http://www.goodreads.com/quotes/456465-float-like-a-butterfly-sting-like-a-bee-the-hands